pmoves/tools/secrets_funnel_populate.py

```
@@ -0,0 +1,845 @@
+from pathlib import Path
+from typing import Any, Optional
+
+
+# ---------------------------------------------------------------------------
+# Module exports
+# ---------------------------------------------------------------------------
+__all__ = [
+    "KeyEntry",
+    "KeyStatus",
+    "ProviderCatalog",
+    "SecretsFunnel",
```

Ruff (0.15.20)

[error] Undefined name `SecretsFunnel` in `__all__`

(F822)